e2e_tests/account_test.py

```diff
@@ -0,0 +1,48 @@
+import os
+import unittest
+from datetime import datetime, date, timedelta
+from api.unit import Unit
+from models.account import *
+
+
+class AccountE2eTests(unittest.TestCase):
+    token = os.environ.get("token")
+    client = Unit("https://api.s.unit.sh", token)
+
+    def test_create_deposit_account(self):
+        request = CreateDepositAccountRequest("checking", {"customer": Relationship("customer","49423")}, {"purpose": "checking"})
```

Comment:
you can't use hard coded values like that, what if someone else not from your org will try to run those tests?
he doesn't have access to this customer.

create an approved application which will automatically creates a customer (there is a method that runs once in a testcase calld setup so this is a good place to create the customer)

then use it instead.

Reply:
thanks for the feedback, I've fixed it